src/backgroundQueue/ProblemType.ts

```diff
@@ -0,0 +1,4 @@
+export enum ProblemType {
```

Comment:
Maybe call this QueueProblemType?

Reply:
I considered that, but this is inside `src/backgroundQueue` and there can't be further problem types.

Reply:
Note that we have to return the same HTTP response, even if the sink refuses the input.

Reply:
Actually, on reflection, it'd indeed be better to call it that, for when one looks it up on the IDE.

Reply:
Change made